legend-engine-xt-sql-postgres-server/src/main/java/org/finos/legend/engine/postgres/auth/KerberosIdentityProvider.java

```diff
@@ -26,4 +28,9 @@ public Identity getIdentityForPassword(String userName, SecureString passwd)
     {
         return DefaultIdentityFactory.INSTANCE.makeIdentity(SubjectTools.getSubjectFromUsernamePassword(userName, passwd.getChars()));
     }
+
+    public static Identity getIdentityForSubject(Subject subject)
+    {
+        return DefaultIdentityFactory.INSTANCE.makeIdentity(subject);
```

Comment:
We should not use `DefaultIdentityfactory` but rather use `IdentityFactoryProvider`.  This will respect whatever implementation is set by the application.

Reply:
Makes sense. This has been updated